second-order-odes/variation-of-parameters.md

```diff
@@ -1,5 +1,53 @@
 # Variation of parameters
 
+What if the ODE does not have constant coefficients or r is not in the table of known functions?
+
+If $y'' + p(x)y' + q(x)y = r(x)$ and $y_h = C_1y_1 + C_2y_2$ solves the homogenous ODE $y'' + p(x)y' + q(x)y = 0$, then 
+
+\begin{align}
+y_p &= -y_1\int\fract{y_2r}{W}dx + y_2\int\fract{y_1r}{W}dx \\
+W &= y_1y_2' -y_2y_1'
+\end{align}
+
+```{topic} W - "Wronksian"
+W is the Wronskian determinant
+```
+
+```{example}
```

Comment:
You can remove this `{example}` directive, and you should get a preview to build.